src/boilerplates/create_embedded_wallet_verify_session.ts

```diff
@@ -0,0 +1,115 @@
+
+import { input } from "@inquirer/prompts";
+import { Command } from "commander";
+
+import shell from "shelljs";
+
+
+const EMBEDDED_WALLET_VERIFY_SESSION_REPO_URL = "https://github.com/0xsequence-demos/embedded-wallet-verify-session";
+
+
+export async function createEmbeddedWalletVerifySession(program: Command, options: any) {
+    let waasConfigKey = options.waasConfigKey;
+    let projectAccessKey = options.projectAccessKey;
+    let googleClientId = options.googleClientId;
+    let builderApiId = options.builderApiId;
+
+    if (!waasConfigKey) {
+        console.log("Please provide the WaaS Config Key for your project.");
+        console.log("Your config key can be found at https://sequence.build under the embedded wallet settings.");
+        console.log("To skip and use the default test config key, press enter.");
+ 
+        waasConfigKey = await input({
+            message: "WaaS Config Key:",
+        });
+
+        console.log("");
+    }
+
+    if (!projectAccessKey && waasConfigKey != '') {
+        console.log("Please provide the Project Access Key for your project.");
+        console.log("Your access key can be found at https://sequence.build under the project settings.");
+        console.log("To skip and use the default test access key, press enter.");
+ 
+        projectAccessKey = await input({
+            message: "Project Access Key:",
+        });
+
+        console.log("");
+    }
+
+    if (!googleClientId && waasConfigKey != '') {
+        console.log("Please provide the Google Client ID for your project.");
+        console.log("Your client ID can be found at https://console.cloud.google.com/apis/credentials");
+        console.log("To skip and use the default test client ID, press enter.");
+ 
+        googleClientId = await input({
+            message: "Google Client ID:",
+        });
+
+        console.log("");
+    }
+
+    if (!builderApiId && waasConfigKey != '') {
+        console.log("Please provide your project id for your project.");
+        console.log("Your project id can be found at https://sequence.build/");
+        console.log("Please select a project and copy your project id from URL. It will look like: 2317");        
+        console.log("To skip and use the default test client ID, press enter.");
+        const baseBuilderApiIdUrl = "https://api.sequence.build/project/";
+        builderApiId = await input({
+            message: "Builder API ID:",
+        });
+
+        if (builderApiId) {
+            builderApiId = baseBuilderApiIdUrl + builderApiId;
+        }
+
+        console.log("");
+    }
```

Comment:
@taylanpince what do you think? Let me know if I should change it. If you can attach me where to get the direct project api url, great.